data/mainnet/tokens.js

```diff
@@ -8854,4 +8854,151 @@ module.exports = {
     rowTooltip: 'Out of Range',
     disableAutoSwap: true,
   },
+  aura_auraBAL: {
```

Comment:
Move these entries up to the top of the file, just below the wETH entry. So they show up at the top of the website

Reply:
Fixed